packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.2.4
+
++ Add `toString` to the genereated class
```

Comment:
Make sure you check out the style guide for plugins/packages. https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style

Reply:
I have adjusted this as the style guide requires.